index.html

```diff
@@ -114,7 +114,80 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-		
+		 <!-- ================================================ -->
+        <!-- ==================  TEMPLATE  ================== -->
+        <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
+ <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```

Comment:
edit this line and put raghav's card start  instead of the default world "template: make a copy contr..."

Reply:
thanks alot